plover/config.py

```diff
@@ -85,6 +85,8 @@
 DEFAULT_TRANSLATION_FRAME_X = -1
 TRANSLATION_FRAME_Y_OPTION = 'y'
 DEFAULT_TRANSLATION_FRAME_Y = -1
+TRANSLATION_FRAME_TRANSPARENCY_OPTION = 'transparency'
```

Comment:
Darn, I thought I got them all. I'll do an ag transparen pass over the source tree.

--
Jeremy W. Sherman
http://jeremywsherman.com/

> El 24-02-2016, a las 17:49, Benoit Pierre <notifications@github.com> escribió:
> 
> In plover/config.py:
> 
> > @@ -85,6 +85,8 @@
> >  DEFAULT_TRANSLATION_FRAME_X = -1
> >  TRANSLATION_FRAME_Y_OPTION = 'y'
> >  DEFAULT_TRANSLATION_FRAME_Y = -1
> > +TRANSLATION_FRAME_TRANSPARENCY_OPTION = 'transparency'
> This should really be renamed to "opacity" too.
> 
> —
> Reply to this email directly or view it on GitHub.
>